src/ext/background.js

```diff
@@ -207,10 +215,44 @@ function processHeaders(details, url) {
  */
 function beforeSendHeaders(request, url) {
     // Cancel if we don't have a token to spend
-    if (!DO_REDEEM || !getSpendFlag(url.host) || checkMaxSpend(url.host) || spentUrl[url.href] || isErrorPage(url.href) || isFaviconUrl(url.href) || REDEEM_METHOD != "reload") {
-        return {cancel: false};
+
+    let reqUrl = url.href;
+    let host = url.host
```

Comment:
for consistency - let's use semicolons at the end

Reply:
For sure, that is a typo